src/DB/exerciseData.json

```diff
@@ -365,5 +365,13 @@
   "videoLink": "https://youtu.be/7jqi2qWAUJk",
   "gh-name": "Kruthardh11",
   "image": "https://cdn.shopify.com/s/files/1/1876/4703/files/shutterstock_419477203_480x480.jpg?v=1636560233"
+},
+
+{
+  "exercise":"lying-leg-curl" ,
```

Comment:
Could you change the name of the exercise to be without dashes?

Reply:
Yeah Sure!